conda/gateways/connection/download.py

```diff
@@ -35,6 +39,42 @@ def download(
 ):
     if exists(target_full_path):
         maybe_raise(BasicClobberError(target_full_path, url, context), context)
+    if sys.platform == 'win32':
+        libbin_path = os.path.join(sys.prefix, 'Library', 'bin')
+        libssl_dllname = 'libssl'
+        libcrypto_dllname = 'libcrypto'
+        libssl_version = '-1_1'
+        libssl_arch = ''
+        if sys.maxsize > 2**32:
+            libssl_arch = '-x64'
+        so_name = libssl_dllname + libssl_version + libssl_arch
+        libssl_path2 = os.path.join(libbin_path, so_name)
+        # if version 1.1 is not found, try to load 1.0
+        if not exists(libssl_path2 + ".dll"):
+            # print("prior not exits? %s\n" % libssl_path2+".dll")
+            libssl_version = '-1_0'
```

Comment:
unfortunately, I don't think this is sufficient.  The openssl 1.0 DLL names were totally different.  Probably best to look at one of the 1.0.2 package contents, but I remember something like libeay32.dll and libssleay32.dll?

Reply:
Sorry, you are right. Just searching for difference of names for x64 and the 32-bit variant